projects/igniteui-angular/src/lib/combo/combo.component.ts

```diff
@@ -42,6 +42,38 @@ export interface IComboFilteringOptions {
     caseSensitive: boolean;
 }
 
+export interface IComboFilteringStrategy {
```

Comment:
Do we need to export an interface and expose the strategy through a wrapping interface?

Reply:
It seems like an unnecessary wrapper, knowing the fact that it contains only one method. Two reasons for having it in that way were to be consistent with the grid, and the other is to be extensible for the future, without introducing a breaking change. But a method is also extensible, knowing the fact that you can add an additional parameter, and we can choose simplicity over consistency with the grid. @wnvko will proceed with exposing a function, @kdinev we just need your approval for its name.